.gitignore

```diff
@@ -1,3 +1,6 @@
+# IDE Files
```

Comment:
I removed my ignore changes, however I now also have edits to project files because I helped by marking some of the `/dist` folders as excluded. I personally think .idea folders should not be checked in and you should use other build tools to have a consistent style guide, but that can be a separate PR

Reply:
SGTM